src/main/java/com/webonise/tomcat8/redisession/RedisSessionManager.java

```diff
@@ -948,4 +994,85 @@ public int getSessionsPerMinute() {
     }
 
   }
+
+public String getHost() {
+	return host;
+}
+
+public void setHost(String host) {
+	this.host = host;
+}
+
+public int getPort() {
+	return port;
+}
+
+public void setPort(int port) {
+	this.port = port;
+}
+
+public int getDatabase() {
+	return database;
+}
+
+public void setDatabase(int database) {
+	this.database = database;
+}
+
+public String getPassword() {
+	return password;
+}
+
+public void setPassword(String password) {
+	this.password = password;
+}
+
+public int getTimeout() {
+	return timeout;
+}
+
+public void setTimeout(int timeout) {
+	this.timeout = timeout;
+}
+
+
+@Override
+protected String getDomainInternal() {
+	// TODO Auto-generated method stub
+	return null;
+}
+
+
+@Override
+protected String getObjectNameKeyProperties() {
+	// TODO Auto-generated method stub
+	return null;
+}
+
+/**
+ * Initial redis config
+ */
+@Override
+protected void startInternal() throws LifecycleException {
+	InitialContext initialContext = null;
+     try {
+         initialContext = new InitialContext();
+         redis = (Redis) initialContext.lookup("java:/comp/env/pool/RedisConfig");
+     } catch (NamingException e) {
+         LOG.warn("RedisConfig not found in JNDI");
+     }
+     if (redis == null) {
+         redis =  new Redis(getHost(),getPort(),getTimeout()); // default configuration
```

Comment:
Don't do this. Throw a LifecycleException or a RuntimeException instead of using some default. You'll make it pretend like it's working.